membership-attribute-service/test/acceptance/AccountControllerAcceptanceTest.scala

```diff
@@ -202,7 +201,6 @@ class AccountControllerAcceptanceTest extends AcceptanceTest {
       subscriptionServiceMock.current(contact)(any) was called
       zuoraRestServiceMock.getGiftSubscriptionRecordsFromIdentityId("200067388")(any) was called
       subscriptionServiceMock.get(Subscription.Name(giftSubscription.Name), isActiveToday = false)(any) was called
-      catalogServiceMock.unsafeCatalog was called
```

Comment:
I would have thought it would only be important to check that side effecting calls were callled or not called?  unless we thought this would be expensive if called unnecessarily - but we only read the catalog once on startup.